test/error/error.test.js

```diff
@@ -1,10 +1,11 @@
-'use strict';
+'use strict'
 
-var wows = require('vows');
-var DOMParser = require('../../lib/dom-parser').DOMParser;
+var wows = require('vows')
+var DOMParser = require('../../lib/dom-parser').DOMParser
 const assert = require('../assert')
 
-var xml = '<scxml xmlns="http://www.w3.org/2005/07/scxml" version="1.0"\n\
+var xml =
+	'<scxml xmlns="http://www.w3.org/2005/07/scxml" version="1.0"\n\
        profile="ecmascript" id="scxmlRoot" initial="start">\n\
```

Comment:
This is the case I was talking about before. But it looks like Prettier is smart enough to not touch these spaces.